lib/aws-record/record/item_operations.rb

```diff
@@ -81,6 +81,24 @@ def save(opts = {})
         end
       end
 
+
+      # Performs mass assignment of model attributes
+      def assign_attributes(opts)
+        opts.each do |field, new_value|
+          if !respond_to?("#{field}=")
+            raise ArgumentError.new "Invalid field: #{field} for model"
+          end
+          @data.set_attribute field, new_value
```

Comment:
Style point in this repo, we explicitly use `()` characters around any non-empty method calls. So:

```ruby
@data.set_attribute(field, new_value)
```